can't test because can't find contrib

After running `make install` I manage to build Idris and seemingly all the libs however when it gets to testing I get 
```
➜  Idris2 git:(2a0e5a4) ✗ make test
idris --build tests.ipkg
Entering directory `./tests'
The following packages were specified but cannot be found:
- contrib
make: *** [test] Error 1
```

I did `make clean && make install` again to make sure and to show the end of the logs here which shows that `contrib` is built
```
network library tests SUCCESS
make -C libs/contrib IDRIS2=../../idris2
../../idris2 --build contrib.ipkg
1/13: Building Syntax.WithProof (Syntax/WithProof.idr)
2/13: Building Syntax.PreorderReasoning (Syntax/PreorderReasoning.idr)
3/13: Building Data.List.TailRec (Data/List/TailRec.idr)
4/13: Building Data.Bool.Extra (Data/Bool/Extra.idr)
5/13: Building Data.SortedMap (Data/SortedMap.idr)
6/13: Building Data.SortedSet (Data/SortedSet.idr)
7/13: Building Text.Token (Text/Token.idr)
8/13: Building Text.Quantity (Text/Quantity.idr)
9/13: Building Control.Delayed (Control/Delayed.idr)
10/13: Building Text.Parser.Core (Text/Parser/Core.idr)
11/13: Building Text.Parser (Text/Parser.idr)
12/13: Building Text.Lexer.Core (Text/Lexer/Core.idr)
13/13: Building Text.Lexer (Text/Lexer.idr)
idris --build tests.ipkg
Entering directory `./tests'
The following packages were specified but cannot be found:
- contrib
make: *** [test] Error 1
```

--- a/tests.ipkg
+++ b/tests.ipkg
@@ -1,7 +1,5 @@
 package runtests
 
-pkgs = contrib
-
 sourcedir = tests
 executable = runtests
 

